<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-14T13:04:44 IST" name="Parallel Testing" finished-at="2025-02-14T13:04:48 IST" duration-ms="4648">
    <groups>
      <group name="smoke">
        <method signature="LoginTest.userLoginUsingValidUsernameAndValidPassword()[pri:1, instance:testScript.LoginTest@367ffa75]" name="userLoginUsingValidUsernameAndValidPassword" class="testScript.LoginTest"/>
        <method signature="LoginTest.userLoginUsingValidUsernameAndInvalidPassword()[pri:2, instance:testScript.LoginTest@15ca7889]" name="userLoginUsingValidUsernameAndInvalidPassword" class="testScript.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2025-02-14T13:04:44 IST" name="Test1" finished-at="2025-02-14T13:04:48 IST" duration-ms="4561">
      <class name="testScript.LoginTest">
        <test-method is-config="true" signature="initializeBrowser(java.lang.String)[pri:0, instance:testScript.LoginTest@367ffa75]" started-at="2025-02-14T13:04:44 IST" name="initializeBrowser" finished-at="2025-02-14T13:04:48 IST" duration-ms="3959" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeBrowser -->
        <test-method signature="userLoginUsingValidUsernameAndValidPassword()[pri:1, instance:testScript.LoginTest@367ffa75]" started-at="2025-02-14T13:04:48 IST" name="userLoginUsingValidUsernameAndValidPassword" description="Login using valid credentials" finished-at="2025-02-14T13:04:48 IST" duration-ms="281" status="FAIL">
          <exception class="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException">
            <message>
              <![CDATA[No valid entries or contents found, this is not a valid OOXML (Office Open XML) file]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:159)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:94)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:157)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:351)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:64)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:313)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:287)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testScript.LoginTest.userLoginUsingValidUsernameAndValidPassword(LoginTest.java:14)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0x7e8dfae2
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:770)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:150)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLoginUsingValidUsernameAndValidPassword -->
        <test-method is-config="true" signature="driverCloseAndQuit(org.testng.ITestResult)[pri:0, instance:testScript.LoginTest@367ffa75]" started-at="2025-02-14T13:04:48 IST" name="driverCloseAndQuit" finished-at="2025-02-14T13:04:48 IST" duration-ms="277" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=userLoginUsingValidUsernameAndValidPassword status=FAILURE method=LoginTest.userLoginUsingValidUsernameAndValidPassword()[pri:1, instance:testScript.LoginTest@367ffa75] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverCloseAndQuit -->
      </class> <!-- testScript.LoginTest -->
    </test> <!-- Test1 -->
    <test started-at="2025-02-14T13:04:44 IST" name="Test2" finished-at="2025-02-14T13:04:48 IST" duration-ms="4648">
      <class name="testScript.LoginTest">
        <test-method is-config="true" signature="initializeBrowser(java.lang.String)[pri:0, instance:testScript.LoginTest@15ca7889]" started-at="2025-02-14T13:04:44 IST" name="initializeBrowser" finished-at="2025-02-14T13:04:48 IST" duration-ms="3769" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeBrowser -->
        <test-method signature="userLoginUsingValidUsernameAndInvalidPassword()[pri:2, instance:testScript.LoginTest@15ca7889]" started-at="2025-02-14T13:04:48 IST" name="userLoginUsingValidUsernameAndInvalidPassword" description="Login using valid username and invalid password" finished-at="2025-02-14T13:04:48 IST" duration-ms="496" status="FAIL">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Stream Closed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Stream Closed
at java.base/java.io.FileInputStream.available0(Native Method)
at java.base/java.io.FileInputStream.available(FileInputStream.java:494)
at java.base/java.io.BufferedInputStream.implRead(BufferedInputStream.java:428)
at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:399)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:119)
at java.base/java.io.PushbackInputStream.read(PushbackInputStream.java:197)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:95)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.fill(ZipArchiveInputStream.java:627)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.readFromInflater(ZipArchiveInputStream.java:1146)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.readDeflated(ZipArchiveInputStream.java:1098)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.read(ZipArchiveInputStream.java:1028)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:119)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.read(ZipArchiveThresholdInputStream.java:78)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:245)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:208)
at org.apache.poi.openxml4j.util.ZipArchiveFakeEntry.<init>(ZipArchiveFakeEntry.java:83)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:98)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:157)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:351)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:64)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:313)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:287)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testScript.LoginTest.userLoginUsingValidUsernameAndInvalidPassword(LoginTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLoginUsingValidUsernameAndInvalidPassword -->
        <test-method is-config="true" signature="driverCloseAndQuit(org.testng.ITestResult)[pri:0, instance:testScript.LoginTest@15ca7889]" started-at="2025-02-14T13:04:48 IST" name="driverCloseAndQuit" finished-at="2025-02-14T13:04:48 IST" duration-ms="341" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=userLoginUsingValidUsernameAndInvalidPassword status=FAILURE method=LoginTest.userLoginUsingValidUsernameAndInvalidPassword()[pri:2, instance:testScript.LoginTest@15ca7889] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverCloseAndQuit -->
      </class> <!-- testScript.LoginTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Parallel Testing -->
</testng-results>
